post-install.yml

```diff
@@ -0,0 +1,168 @@
+- hosts: "{{ hosts_prefix }}-tsuru-api*[0]"
+  sudo: yes
+  tasks:
+    - name: Check tsuru_deployer key
+      stat: path=~/.ssh/tsuru_deployer
+      register: tsuru_deployer_key
+      delegate_to: "localhost"
+      sudo: no
+
+    - name: Generate tsuru_deployer key
+      shell: >
+        ssh-keygen -b 2048 -t rsa -f ~/.ssh/tsuru_deployer -q -N ""
+      when: tsuru_deployer_key.stat.exists == False
+      delegate_to: "localhost"
+      sudo: no
+    - name: Generate tsuru_deployer.pub key
```

Comment:
This feels fairly intrusive, can we use some other way? Generating keys on my laptop for remote applications with out my approval feels like we are walking in root kit territory.